discovery/discovery.go

```diff
@@ -207,6 +214,24 @@ func (o *orchestratorPool) GetOrchestrators(ctx context.Context, numOrchestrator
 	return ods, nil
 }
 
+func sendLiveAICapacityMetrics(caps *net.Capabilities) {
+	if !monitor.Enabled {
+		return
+	}
+	if caps == nil || caps.Capacities == nil || caps.Constraints.PerCapability == nil {
+		return
+	}
+	liveAI, ok := caps.Constraints.PerCapability[uint32(core.Capability_LiveVideoToVideo)]
+	if !ok {
+		return
+	}
+
+	for _, model := range liveAI.Models {
+		monitor.AIContainersInUse(int(model.CapacityInUse))
```

Comment:
@leszko Only thing I'm unsure of is whether to re-use these metrics with a new tag or create entirely new ones? I'm thinking re-use them and distinguish them with a `gateway: true` tag in the interim while we switch to using this method rather than Os reporting it. Because I think then we would just remove the Os reporting the metrics, this PR will just replace that.